meta-lmp-bsp/recipes-bsp/u-boot/u-boot-base-scr.bbappend

```diff
@@ -1 +1,11 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+do_install:append:stm32mp15-eval() {
+    install -m 0644 ${DEPLOYDIR}/boot.scr ${D}/boot.scr
+    install -m 0644 ${DEPLOYDIR}/uEnv.txt ${D}/uEnv.txt
+}
+
+FILES:${PN} += " \
```

Comment:
the FILES needs the same override `FILES:${PN}:stm32mp15-eval` otherwise will fail as this files not found in other machines

Reply:
sorry, missed that. Will address in another PR

Reply:
@quaresmajose @igoropaniuk we have a generic version of boot.cmd and uEnv.txt.in which deploys as boot.scr and uEnv.txt, so I think it's safe to keep a common FILES instruction.

Reply:
I am testing this MR now in https://github.com/foundriesio/lmp-manifest/pull/221 so in a couple of hours we will know.

Reply:
@MrCry0 is right and we don't need `FILES:${PN}:stm32mp15-eval` override as MR https://github.com/foundriesio/lmp-manifest/pull/221 build without any issue.
@igoropaniuk however, this way I think we don't need the full block of `FILES:${PN}`, tomorrow I will test

Reply:
We always deploy both boot.scr and uEnv.txt, so I think it is fine to create a generic do_install function that install both, instead of having it just for stm32mp15-eval.

Then we can both everything to meta-lmp-base (from the bbappend).

Reply:
@ricardosalveti but should we have both (boot.scr and uEnv.txt) in rootfs? For most platforms they are installed via WKS parser and `IMAGE_BOOT_FILES`.

Reply:
I think it is fine to have them in the rootfs, even if they are also listed in IMAGE_BOOT_FILES because that one is only for copying into the vfat/boot partition.